Modix/Modules/InfractionModule.cs

```diff
@@ -76,6 +78,74 @@ public InfractionModule(IModerationService moderationService)
             if(replyBuilder.Length > 0)
                 await ReplyAsync(Format.Code(replyBuilder.ToString()));
         }
+        
+        [Command("search embed")]
```

Comment:
We may migrate this to be the only version of this command, but good to keep it separate for now, for testing.